docs/whats_new/version_0_23_0.rst

```diff
@@ -29,8 +29,13 @@ Add changeable logger class for the test
 Add the possibility to change the logger class with the CLI.
 For more information please refer to :ref:`change_logger_class`.
 
-Add shutdown method to CCHannel 
+Add shutdown method to CCHannel
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Add proper shutdown method for connector instead of instead of using
 the del method
+
```

Comment:
Something is strange here. We are releasing 0.24.0 but we added information to 0.23.0 rst file. In addition, I think we forgot to create and fill the `version_ongoing.rst` with any contribution done after 0.23.0

Reply:
The problem is that there is nothing noticeable to add to the release, but we need to go through because we need to release the internal part